### ISSUE TITLE
Loss is NaN and the acc = ~0.10

### ISSUE THREAD

**Issue by jeffreyflukman** [geifmany/cifar-vgg#1]
Hi Geifmany,

I tried your vanilla code here to understand how Keras works.
But when I run it, the model experience loss : nan and acc: ~0.10

Did you also experience it?
If not, is there any parameter that you tune?

**Comment by jeffreyflukman** [comment]
I have changed the learning rate to be 0.01 and then it works well.
